### ISSUE TITLE
can’t be finished using learn/learn submit

### ISSUE THREAD

**Issue by mikemerin** [learn-co-curriculum/creating-and-inserting-dom-nodes#10]
I don’t know if this is the issue with it, but they have reading icons but require local tests/submit pull request, and the lesson can't do either of them.

**Comment by edmundo096** [comment]
Same here.
Doing `learn open creating-and-inserting-dom-nodes-cb-gh-00` didn't do the trick:

> Looking for lesson...
> It looks like this lesson is a Readme. Please open it in your browser.

**Comment by edmundo096** [comment]
This happens as well on https://github.com/learn-co-curriculum/moving-things-with-javascript

I found a workaround meanwhile...
I forked it, cloned it into the _labs_ directory.
Then manually created the test files, or copy the test directory from other JS lab which uses the same assertion library, in this case, one that has *chai* on the _package.json_. If you want, create the corresponding .js files, e.g. the _index.js_ and _index-test.js_.
Make it pass with `learn`, and then `learn submit` it.
